### PR TITLE
Add ECS Fargate storage docs

### PR DESCRIPTION
**Pull request by mabadir**
This add docs related to how to configure the Fargate storage in `fargate` and `fargate-worker` services.
Also adds troubleshooting document for error
```
[
  {
    "type": "FARGATE_DEPLOY_ERROR",
    "message": "Cannotpullcontainererror: ref pull has been retried 1 time(s): failed to extract layer: <layer_id> no space left on device: unknown"
  }
]
```

Also adds troubleshooting docs for:
1. Increasing Node memory
2. Customizing CodeBuild configuration

### CODE REVIEW

File: motif-docs/pages/troubleshooting/increase-node-memory.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+---
+
+If the user is getting this error during build or Runtime:
+
+```
+Reached heap limit Allocation failed - JavaScript heap out of memory
+```
+
+1. This error means the Node application is running out of memory.
+2. Add the following environment variable `NODE_OPTIONS=--max-old-space-size=4096` which allocates 4GB of memory to the node process.
```
Comment (flybayer) [inline motif-docs/pages/troubleshooting/increase-node-memory.mdx:13]: add a note + link to other page about increasing codebuild memory?

File: motif-docs/pages/troubleshooting/configure-codebuild.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import Image from 'next/image'
+import buildLogsButtonImg from '../../public/img/view-build-aws-logs-button.png'
+import codeBuildEditEnv from '../../public/img/codebuild-edit-environment.png'
+import codeBuildEditEnvAdditionalConfig from '../../public/img/codebuild-edit-environment-additional-config.png'
+
+Flightcontrol provisions a CodeBuild project for every service that requires build, this allows different services to build concurrently.
+The standard provisioned project environment (7 GB memory, 4 vCPUs) should be suitable for almost all use cases. If however there is a need to change this configuration:
+
+1. Allocate the CodeBuild project, the easiest way is to click the **View Logs in AWS** button in any Build Logs dialogue.
+<Image className="rounded-primary mx-auto w-full max-w-[250px] border" src={buildLogsButtonImg} />
```
Comment (flybayer) [inline motif-docs/pages/troubleshooting/configure-codebuild.mdx]: can you make this smaller, and add an arrow pointing to the view logs button?

File: motif-docs/pages/troubleshooting/configure-codebuild.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+2. This will open the CodeBuild project, You will need to go one level up in the project breadcrumps.
+3. Find the project Edit button, then click **Edit Environment**
+<Image className="rounded-primary mx-auto w-full max-w-[250px] border" src={codeBuildEditEnv} />
```
Comment (flybayer) [inline motif-docs/pages/troubleshooting/configure-codebuild.mdx]: make this smaller too, to match the actual size

File: motif-docs/pages/troubleshooting/configure-codebuild.mdx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+4. Expand the **Additional configuration** section, which will allow you to choose the environment configuration required.
+<Image className="rounded-primary mx-auto w-full max-w-[250px] border" src={codeBuildEditEnvAdditionalConfig} />
```
Comment (flybayer) [inline motif-docs/pages/troubleshooting/configure-codebuild.mdx]: also smaller

File: motif-docs/pages/troubleshooting/increase-node-memory.mdx (hunk 1 above)
Comment (mabadir) [reply motif-docs/pages/troubleshooting/increase-node-memory.mdx:13]: > add a note + link to other page about increasing codebuild memory?

Added

File: motif-docs/pages/troubleshooting/configure-codebuild.mdx (hunk 2 above)
Comment (mabadir) [reply motif-docs/pages/troubleshooting/configure-codebuild.mdx]: > can you make this smaller, and add an arrow pointing to the view logs button?

Done

File: motif-docs/pages/troubleshooting/configure-codebuild.mdx (hunk 3 above)
Comment (mabadir) [reply motif-docs/pages/troubleshooting/configure-codebuild.mdx]: Done

File: motif-docs/pages/troubleshooting/configure-codebuild.mdx (hunk 4 above)
Comment (mabadir) [reply motif-docs/pages/troubleshooting/configure-codebuild.mdx]: Done

**Review by flybayer — APPROVED** [review]

**Pull request merged**